src/db/dbQuery.ts

```diff
@@ -0,0 +1,20 @@
+import { database } from "./dbConfig";
+
+
+export const getAllUnsentEmailData =() => {
+    try {
+        return new Promise((resolve, reject) => {
+            database.query('select * from notification where email=true and email_sent=false and seen=false;', function (err, result){
+                if (err){
+                    console.log("Garden Error: " + err);
```

Comment:
What is garden error?

Reply:
Actually! This is just the base structure I created In the next commits I will do the things required for this issue 

Currently The Status of this PR is in-progress so you can suggest me if any of the thing Improve I will change the status to DONE when its completed

Reply:
Sorry for the misunderstanding. 
My approach would have been to mark the PR  as a draft and put some TODOS, but anyway each person has their own style.